dev-packages/application-manager/src/generator/frontend-generator.ts

```diff
@@ -140,6 +140,7 @@ const { ElectronSecurityToken } = require('@theia/core/lib/electron-common/elect
 const applicationName = \`${this.pck.props.frontend.config.applicationName}\`;
 const isSingleInstance = ${this.pck.props.backend.config.singleInstance === true ? 'true' : 'false'};
 const disallowReloadKeybinding = ${this.pck.props.frontend.config.disallowReloadKeybinding === true ? 'true' : 'false'};
+const borderless = ${this.pck.props.frontend.config.borderless === true ? 'true' : 'false'};
```

Comment:
A better name for this option would be [frameless](https://www.electronjs.org/docs/api/frameless-window#frameless-window) like in the documentation.

Reply:
Done 👍